### PR TITLE
Fixed typo in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by toir427**
Fixed typo in CONTRIBUTING.md

- `want contributing` -> `want to contribute`
- `there` -> `then`
- `wont` -> `won't`

### CODE REVIEW

**Comment by toir427** [comment]
`spcific` -> `specific`

**Comment by toir427** [comment]
I think you must use Appwrite not the Appwrite [info](https://www.google.com/search?q=can+i+use+the+article+before+organization+name&oq=can+i+use+the+article+before+organization+name&aqs=chrome..69i57j69i64.11361j1j7&sourceid=chrome&ie=UTF-8)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-We would ❤️ for you to contribute to Appwrite and help make it better! We want contributing to Appwrite to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, new docs as well as updates and tweaks, blog posts, workshops, and more.
+We would ❤️ for you to contribute to Appwrite and help make it better! We want to contribute to Appwrite to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, new docs as well as updates and tweaks, blog posts, workshops, and more.
```
Comment (stnguyen90) [inline CONTRIBUTING.md:3]: This was correct as is.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ When `TYPE` can be:
 
 For the initial start, fork the project and use git clone command to download the repository to your computer. A standard procedure for working on an issue would be to:
 
-1. `git pull`, before creating a new branch, pull the changes from upstream. Your master needs to be up to date.
+1. `git pull`, before creating a new branch, pull the changes from upstream. Your master needs to be up-to-date.
```
Comment (stnguyen90) [inline CONTRIBUTING.md:37]: This was correct as is. See: https://writingexplained.org/up-to-date-hyphenated

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
@toir427 Great PR! 🤯 We left some comments during the review, please check them out.

**Comment by stnguyen90** [comment]
Closing stale PR.

**Pull request closed without merging**